tests/test_filter.py

```diff
@@ -0,0 +1,18 @@
+from kwalitee.release import get_releases
+from kwalitee.filter import filter_release_projects
+
+def test_filter():
+    releases = get_releases(test_mode=True)
+    filtered = filter_release_projects(releases)
+    release = filtered[1]
+    assert release['projects'] == ['eynollah', 'ocrd_cis', 'ocrd_doxa', 'ocrd_fileformat', 'sbb_binarization']
+
+def test_ignore_releases_without_project_after_filtering():
+    releases = get_releases(test_mode=True)
+    filtered = filter_release_projects(releases)
+    is_tag_in_list = False
+    for entry in filtered:
+        # an example of an empty release (i.e. one without projects) after filtering is v2022-02-03
```

Comment:
Why is https://github.com/OCR-D/ocrd_all/releases/tag/v2022-02-03 an "empty release"? It does contain changes to eynollah, ocrd_cis and many more and even adds two new projects ocrd_doxa and ocrd_detectron2.

Reply:
The way we designed the new dashboard requires that each project is listed only once – in the latest/most recent ocrd_all release it is part of.

`v2022-02-03` contains the following projects:

- core  (latest ocrd_all release: v2022-06-03)
- tesseract  (latest ocrd_all release: v2022-03-20)
- ocrd_cis  (latest ocrd_all release: v2022-05-04)
- cor-asv-ann  (latest ocrd_all release: v2022-03-25)
- dinglehopper  (latest ocrd_all release: v2022-03-30)
- eynollah  (latest ocrd_all release: v2022-05-04)
- ocrd_fileformat  (latest ocrd_all release: v2022-05-04)
- ocrd_segment  (latest ocrd_all release: v2022-05-04)
- opencv-python  (latest ocrd_all release: v2022-06-03)
- workflow-configuration  (latest ocrd_all release: v2022-06-03)

As you can see, every project that is part of the release is also part of a more recent ocrd_all release. Therefore its `projects` attribute has to be empty after applying the filter.

Reply:
I added a description of the filter with [668228e](https://github.com/OCR-D/kwalitee-dashboard-back-end/pull/7/commits/668228ea030dbd46e54910e2f506aefef597f3a1). I hope this makes things a bit more understandable.

Reply:
I see. I would have expected the grouping/collating to happen in the frontend. But if this approach works better for you and @paulpestov, go for it.